src/components/DetailForm.vue

```diff
@@ -79,7 +79,7 @@ export default createComponent({
     const initialFactories = mapController.value?.factories
     const initialLocation = mapController.value?.mapInstance.map.getView().getCenter()
 
-    const reportTypeItems: Array<{ text: string, value?: number }> = [
+    const reportTypeItems: Array<{ text: string, value?: string }> = [
```

Comment:
Oh, I did not mention this in my commit and description. This issue was found when I tried to keep working on the refactoring task. I think @Neilxx may introduce an issue in his previous refactoring effort.

Reply:
Yes I did. My bad 😅
Thanks for debugging!